app/views/gitter/chat_button.erb

```diff
@@ -0,0 +1,17 @@
+<% if control.language.downcase == 'ruby' %>
+  <div class="gitter-chat-button-frame">
+    <div class="gitter-chat-button">
+      Open Chat: Enjoy the <%= control.language.capitalize %> conversation!
+    </div>
+  </div>
+
+  <script>
+    ((window.gitter = {}).chat = {}).options = {
+      room: 'exercism/xruby',
```

Comment:
I see what you are saying...  I was reading into it that you were saying coming from 'x-api' track data, not the config file.